sick_scan2: # configuration for sick_scan2_mrs_1xxx: For compatibility with ros versions previous to foxy, node and parameter name changed to sick_scan2 for all supported scanner. The type of scanner is configured by scanner_name only.
  ros__parameters:
    hostname : "192.168.0.1"
    frame_id : "cloud"
    imu_frame_id : "test"
    scanner_name : "sick_mrs_1xxx"
    port : 2112
    min_ang : -0.829031395 #in lidar cordinate frame  see ../doc/3d_coordinate_system_comp.png
    max_ang : 3.970624 #in lidar cordinate frame  see ../doc/3d_coordinate_system_comp.png
    max_range : 64.0 #in meters
